### PR TITLE
App homepage query caching

### PR DESCRIPTION
**Pull request by ivanslabbert**

### CODE REVIEW

File: app/src/pages/HomePage/HomePageNewest.vue
Comment (laurensgroeneveld) [inline app/src/pages/HomePage/HomePageNewest.vue:1]: The convention I tried to have is that all components in `pages` are top-level page components, i.e. they get rendered directly in the `RouterView`. These subcomponents for Homepage should then rather belong in `components`.

**Review by laurensgroeneveld — APPROVED** [review]
Just one small comment, but looks good otherwise 👍 I don't have enough content in my local db to test this properly, but I trust you did 😄

File: app/src/pages/HomePage/HomePageNewest.vue
Comment (ivanslabbert) [reply app/src/pages/HomePage/HomePageNewest.vue:1]: Sure, thanks. I can move them.

**Comment by ivanslabbert** [comment]
> Just one small comment, but looks good otherwise 👍 I don't have enough content in my local db to test this properly, but I trust you did 😄

I copied the staging database to my local dev environment to do testing and got good results (using the performance slowdown settings in chrome's dev tools). But will be nice to test on staging in a production environment.

**Pull request merged**